CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Past Releases
 
+## [4.0.2] - 2024-06-04
```

Comment:
Nit: Should this be 4.1.0 given it's a behavioral change? Up to you!

Reply:
makes sense